android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/CountDownTimerFactory.java

```diff
@@ -53,15 +56,22 @@ public static void stopAlarm() {
 
     @Override
     public List<View> getViewsFromJson(String stepName, Context context, JsonFormFragment formFragment, JSONObject jsonObject, CommonListener listener, boolean popup) throws Exception {
-        List<View> views = new ArrayList<>(1);
+        widgetArgs = new WidgetArgs();
+        widgetArgs.withStepName(stepName);
```

Comment:
You can join the `with` assignments into one line

Reply:
Cool. Yeah it will be more readable that way